src/WindowsUtils/CreateProcessTest.cpp

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "OrbitBase/ExecutablePath.h"
+#include "Test/Path.h"
+#include "TestUtils/TestUtils.h"
+#include "WindowsUtils/CreateProcess.h"
+#include "WindowsUtils/ProcessList.h"
+
+namespace {
+std::filesystem::path GetTestExecutablePath() {
+  static auto path = orbit_base::GetExecutableDir() / "FakeCliProgram.exe";
+  return path;
+}
+
+}  // namespace
+
+using orbit_windows_utils::ProcessInfo;
+using orbit_windows_utils::ProcessList;
+
+TEST(CreateProcess, SuccessfulProcessCreation) {
+  constexpr const char* kArguments = "sleep_for_ms=5000";
```

Comment:
Sleeping in tests to "win a race" usually causes trouble when executing them in docker (on the CI). 

If you are sure this works reliably please ignore ... Otherwise maybe you can just put the thing in an endless sleep and kill it when you are done?